docs/samples/calling/app.js

```diff
@@ -209,10 +207,58 @@ function initWebex(e) {
     };
   }
 
-  webex = window.webex = Webex.init(webexConfig);
+  const clientConfig = {
+    calling: true,
+    contact: true,
+    callHistory: true,
+    callSettings: true,
+    voicemail: true,
+  }
+
+  const loggerConfig = {
+    level: 'info'
+  }
+
+  const {region, country} = registrationForm.elements;
+
+  const serviceData = {indicator: 'calling', domain: ''};
+
+  if (serviceIndicator.value) {
+    serviceData.indicator = serviceIndicator.value;
+  }
+
+  if (serviceDomain.value) {
+    serviceData.domain = serviceDomain.value;
+  }
+
+  const callingClientConfig = {
+    logger: loggerConfig,
+    discovery: {
+      region: region.value,
+      country: country.value,
+    },
+    serviceData,
+  };
+
+  if (callingClientConfig.discovery.country === 'Country') {
+    callingClientConfig.discovery.country = '';
+  }
+
+  if (callingClientConfig.discovery.region === 'Region') {
+    callingClientConfig.discovery.region = '';
+  }
+
+  const callingConfig = {
+    clientConfig : clientConfig,
+    callingClientConfig: callingClientConfig,
+    logger:loggerConfig
+  }
+
+  calling = new Calling({options: webexConfig, callingConfig: callingConfig});
+  webex = window.webex = calling.webex;
```

Comment:
when some just want to use the calling functionality do we have to expose the webex object ?

developers should only see the Calling instance and nothing about webex

Reply:
We are not trying exposing the webex object, since it's being imported in calling.js file it is present inside the calling instance. But developer will be using only calling instance only. We will not be documenting this so developer won't be aware if webex exists inside calling instance

Reply:
@Kesari3008 if we are no longer exposing the webex object, do we even need a constant for it? Let's remove all usages of webex object right? The app.js should reflect what the developers will be doing.

Reply:
Taken care. Removed accessing webex object directly from app.js